PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -169,6 +169,11 @@
   btagDeepCvL = Var("?(pt>=15)&&(bDiscriminator('pfDeepCSVJetTags:probc')>=0)?bDiscriminator('pfDeepCSVJetTags:probc')/(bDiscriminator('pfDeepCSVJetTags:probc')+bDiscriminator('pfDeepCSVJetTags:probudsg')):-1", float,doc="DeepCSV c vs udsg discriminator",precision=10),
   btagDeepCvB = Var("?(pt>=15)&&bDiscriminator('pfDeepCSVJetTags:probc')>=0?bDiscriminator('pfDeepCSVJetTags:probc')/(bDiscriminator('pfDeepCSVJetTags:probc')+bDiscriminator('pfDeepCSVJetTags:probb')+bDiscriminator('pfDeepCSVJetTags:probbb')):-1",float,doc="DeepCSV c vs b+bb discriminator",precision=10),
 )
+
```

Comment:
Don't you need to:
`from Configuration.Eras.Modifier_run3_common_cff import run3_common`
?

Reply:
I would have had indeed, sorry about that. What I did now was to use the modifiers that are already imported, in this case the logic changes such that the standard case is to not read CSVV2, and only for the earlier era it is added.